app/src/main/res/menu/starred_list_context_menu.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
:information_source: Please list the generic icon names which are changed in the commit message body so it easier to search for them and get a quick overview.